### PR TITLE
Install heap analytics

### PR DESCRIPTION
**Pull request by adamduren**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -36,6 +36,12 @@
   <script>
     Raven.config('https://e8632a13180c4b218d4deea1c7cf0b26@sentry.io/96050').install();
   </script>
+
```
Comment (zacharytamas) [inline _includes/head.html]: We should really look into async-ing most of this stuff.

File: our-services.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ permalink: /our-services/
       </div>
     </div>
 
-    <button class="btn cta" onclick="contactUs.open()">
+    <button class="btn cta contact-us-btn" onclick="contactUs.open()">
```
Comment (zacharytamas) [inline our-services.md]: Is this used for styling or selection? Don't see styles accompanying this change but they could already exist. We need to organize and split our our Sass.

File: _includes/head.html (hunk 1 above)
Comment (adamduren) [inline _includes/head.html]: If you async you might not catch page load exceptions.

File: our-services.md (hunk 2 above)
Comment (adamduren) [inline our-services.md]: It's used for uniquely identifying buttons or links that open the modal for heap analytics.

File: _includes/head.html (hunk 1 above)
Comment (zacharytamas) [inline _includes/head.html]: Good point. Urgh.

File: our-services.md (hunk 2 above)
Comment (zacharytamas) [inline our-services.md]: Ah so you can see which is most effective?

**Pull request merged**